draft-looker-jwm.xml

```diff
@@ -28,9 +28,9 @@
     <date day="21" month="December" year="2019"/>
 
     <abstract>
```

Comment:
Currently the abstract is also most of the introduction, could you please copy down the text from here into that section too?

Reply:
Done.